pkg/jobs/scheduled_job.go

```diff
@@ -39,11 +39,11 @@ type scheduledJobRecord struct {
 	ScheduleName    string                    `col:"schedule_name"`
 	Owner           string                    `col:"owner"`
 	NextRun         time.Time                 `col:"next_run"`
+	ScheduleStatus  string                    `col:"schedule_status"`
```

Comment:
Even though we're just doing a string _for now_ I'd put a proto message with a string in it here, just so we have flexibility later and avoid annoying schema change migrations. It doesn't need to change what we'd show since we can pb_to_json()->'message' it.